tests/src/JIT/Performance/CodeQuality/HWIntrinsic/X86/PacketTracer/Plane.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+//
+
+internal class Plane : SceneObject
```

Comment:
Why class, instead of a struct?

Wouldn't a struct passed around in a `SceneObject<T>` be more efficient/etc?

Reply:
I will try it based-on the profiling.

Reply:
If we are only ever accessing it in terms of the class it is probably fine... Just seemed a little different to how I would expect the code to be structured